src/main/java/com/softserve/bookstore/service/UserService.java

```diff
@@ -15,17 +16,19 @@
 public class UserService {
 
     @Autowired
-    UserRepository userRepository;
+    private UserRepository userRepository;
+    @Autowired
+    private ManageUserData manageUserData;
 
     public List<User> getAllUsersFromFile(String fileName) throws IOException {
-        List<User> users = userRepository.findAllFromFile(fileName);
+        List<User> users = manageUserData.readDataFromFile(fileName);
```

Comment:
good naming of methods is a key for understanding code
in this case readDataFromFile is much more understandable than findAllFromFile. 
also I would consider using readUserDataFromFile name instead 

FYI: if you have free time consider reading Clean Code by Robert Martin, at least couple of first chapters
in other case you can find some info on the internet about naming conventions